pulpanalytics/models.py

```diff
@@ -8,9 +8,13 @@ class System(models.Model):
     system_id = models.UUIDField()
     created = models.DateTimeField(auto_now_add=True)
     first_seen = models.DateTimeField()
+    postgresql_version = models.TextField()
 
     def __str__(self):
-        return f"SystemID={self.system_id}, Created={self.created}"
+        return (
+            f"SystemID={self.system_id}, Created={self.created}, "
+            f"Postgresql Version={self.postgresql_version}"
```

Comment:
I don't see this change coming. This is supposed to be like the title on a book spine. The postgres version should be available when you open the book (in the detail view). Also is see more attributes to come...

Reply:
That makes sense. I'll remove it in my next push.